Observation/Observation-Necrosis-Example.json

```diff
@@ -0,0 +1,46 @@
+{
+  "resourceType": "Observation",
+  "id": "Observation-Necrosis-Example",
+  "status": "final",
+  "code": {
+    "coding": [
+      {
+        "system": "http://snomed.info/sct",
+        "code": "6574001",
+        "display": "Necrosis (morphologic abnormality)"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-ZelmaHadjkiss-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307555"
+    }
+  },
+  "specimen": {
+    "identifier": {
+      "system": "https://fhir.add.nhs.uk/Id/specimenId",
+      "value": "RA257630"
+    }
+  },
+  "valueQuantity": {
+    "value": 5,
+    "unit": "%",
+    "system": "http://unitsofmeasure.org",
+    "code": "%"
+  },
+  "interpretation": [
```

Comment:
I think cellularity High will need to be a separate observation rather than an interpretation of the necrosis observation